src/tests/topp/MSstatsConverter_3_design.tsv

```diff
@@ -0,0 +1,43 @@
+Fraction_Group	Fraction	Spectra_Filepath	Label	Sample
+1	1	QExactiveHF02_03904.mzML	1	1
+1	1	QExactiveHF02_03904.mzML	2	2
+1	1	QExactiveHF02_03904.mzML	3	3
+1	1	QExactiveHF02_03904.mzML	4	4
+1	1	QExactiveHF02_03904.mzML	5	5
+1	1	QExactiveHF02_03904.mzML	6	6
+1	1	QExactiveHF02_03904.mzML	7	7
+1	1	QExactiveHF02_03904.mzML	8	8
+1	1	QExactiveHF02_03904.mzML	9	9
+1	1	QExactiveHF02_03904.mzML	10	10
+2	1	QExactiveHF02_03905.mzML	1	11
+2	1	QExactiveHF02_03905.mzML	2	12
+2	1	QExactiveHF02_03905.mzML	3	13
+2	1	QExactiveHF02_03905.mzML	4	14
+2	1	QExactiveHF02_03905.mzML	5	15

```

Comment:
it may be helpful for testing this to actually use multiple fractions?

Reply:
Yes, a test with multiple fractions already exists (MSstatsConverter_2_design.tsv):
https://github.com/OpenMS/OpenMS/blob/develop/src/tests/topp/CMakeLists.txt#L1171

The additional test in this PR covers only the special case if one fraction is used, to have a uniform output.